__tests__/lockfile.js

```diff
@@ -193,3 +193,101 @@ test('Lockfile.getLockfile (sorting)', () => {
 
   expect(actual).toEqual(expected);
 });
+
+test('parse single merge conflict', () => {
+  const file = `
```

Comment:
It's much easier to manage it like this, and we care more about the text input than we care about where it is coming from for the purpose of this test.